### ISSUE TITLE
1.0.0: There is no install target

### ISSUE THREAD

**Issue by dvzrv** [AidaDSP/AIDA-X#26]
Hi! When packaging this for a distribution, my expectation is that there is an install target.
However, `DESTDIR="$pkgdir" cmake --install build` unfortunately installs nothing and I have to pick files manually.

**Comment by falkTX** [comment]
correct. something done with a pull request on this repo but I want to integrate that into DPF and some here too.
will do this for perhaps a 1.1 release

**Comment by dvzrv** [comment]
> something done with a pull request on this repo but I want to integrate that into DPF and some here too.

Sounds like a good plan! :)

**Comment by Justin-Hoffman** [comment]
Is this still in a fork and waiting for merge? Main still lacks this as of today.

**Comment by falkTX** [comment]
Correct, I meant to have these changes in DPF rather than each plugin individually, but didnt get to that yet.
Lots of big life changes lately, very little time for opensource things.
Will get back to these things properly at some point...
